ms-engine/src/main/java/net/trajano/ms/engine/internal/resteasy/VertxAsynchronousResponse.java

```diff
@@ -373,7 +392,8 @@ public void setWriterInterceptors(final WriterInterceptor[] writerInterceptors)
      */
     private void writeResponse(final Response response) throws IOException {
 
-        ServerResponseWriter.writeNomapResponse((BuiltResponse) response, request, new VertxHttpResponse(routingContext), providerFactory);
+        ServerResponseWriter.writeNomapResponse((BuiltResponse) response, request, new VertxHttpResponse(routingContext), providerFactory, t -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)